asciinema2gif

```diff
@@ -88,7 +88,7 @@ done
 if [[ -z "${1}" ]] || [[ -n "${2}" ]]; then
   usage
   exit 1
-elif [[ "${1}" =~ ^[[:digit:]]+$ ]]; then
+elif [[ "${1}" =~ ^[[:alnum:]]+$ ]] && [[ $(curl -s -I "${asciinema_api}/${1}" | grep -c 200) == 2 ]]; then
```

Comment:
oh sorry i kinda whipped up the curl part fast without checking the style in the file